astropy/table/table.py

```diff
@@ -374,6 +374,10 @@ class Table:
         Row-oriented data for table instead of ``data`` argument.
     copy_indices : bool, optional
         Copy any indices in the input data. Default is True.
+    units : list, dict, optional
+        List or dict of units to apply to columns
```

Comment:
Missing period at end of line (also below and in the copy in `connect.py`

Reply:
Done.